### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by martin-cervantes1**
## Smashing Magazine Clone

* Complete layout of smashing magazine webpage
* Put the styling gray scale heap map 
* We highlight the important parts of the page with a darker color

### CODE REVIEW

File: index.html
Comment (adaorachi) [inline index.html:408]: - [**optional** ] From this part of your code downwards, you deviated from the use of the semantic element `<section>` to `<div>` for your parent sectional containers. I would love it if you continued with the `section` tag for all the sections so as to make your code uniform and more modular.

**Comment by adaorachi** [comment]
👋 Hi @mcervantes71 and @gabrie-lhilarion,

You both did a nice job on this project. Your Pull Request has a good title and descriptive summary. You made good use of images and adequate use of semantic HTML elements. The elements and images are well aligned and positioned. In all, your project view looks very appealing even without colors. Good Job!👍💯🌟

However, there are a couple of things I will want you to fix before you can get approval for this project.

So, just hang in there, you are just a step away from completing this project.

---
### **CHANGES REQUIRED** ♻️ 
---

- [x] Your linters has a couple of errors. Make sure that your project passes all linter checks.

---

- [x] Also ensure that your HTML markup are properly validated with no errors or warnings. Check your code with this validator - [HTML Validator](https://validator.w3.org/#validate_by_upload) and kindly fix all error/warning issues.

---

- [x]  Your README file looks okay but it should be improved and made more professional by seeing the highlighted areas in the screenshot below.
First, replace the dummy content with a proper description of your project and the features you implemented in the project.
Second, the installation process does not look intuitive enough. Keep in mind that users with no technical background can also make use of your repo. So, ensure your installation processes are detailed like adding a terminal command on how to clone the project, how to cd into the repo and open it on the browser.

smash

---

**The following requirements are considered OPTIONAL and so, are NOT mandatory but I will encourage you to take a moment to look into it as it will be helpful to you in this project and future ones.
NOTE: Requirements NOT marked as OPTIONAL are mandatory!**

---

-[**optional**] In future projects, you should always give your feature branch a descriptive name that shortly describes the project, e.g. smashing_teardown, instead of 'feature_branch' like you have it.

---

**Review by adaorachi — CHANGES_REQUESTED** [review]

**Comment by adaorachi** [comment]
### **After implementing the MANDATORY changes highlighted above, request for another review by submitting a Code Review Form.**

---

Should you have any questions or need any help, please feel free to reach out to me on Slack with the username (@adaorachi).

Happy Coding and Good-luck on it. ☺️

**MaryAnn Chukwuka**

**Comment by SofiBretz** [comment]
### Changes required ♻️
### Hello team 🌟

Great work on this project, but you need to fix the issues below before moving on ⬇️

- [x] As the last TSE required: "...Second, the installation process does not look intuitive enough. Keep in mind that users with no technical background can also make use of your repo. So, ensure your installation processes are detailed like adding a terminal command on how to clone the project, how to cd into the repo and open it on the browser." Consider fixing this on your '.readme' file.

[OPTIONAL]

- [x] Make sure that basic CSS bad practices are not used ([article](https://speckyboy.com/good-bad-css-practices/) for support ) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright) no repeating of the same rule

_**After making the required changes please submit a new code review request**_
Good luck! 🍀 You are almost there! 💪

SofiBretz.

**Comment by martin-cervantes1** [comment]
* Add descriptive installation process 😃

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  TEAM 👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**
